internal/pkg/cli/flag.go

```diff
@@ -104,8 +106,14 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 	envProfilesFlagDescription       = "Optional. Environments and the profile to use to delete the environment."
 	deleteSecretFlagDescription      = "Deletes AWS Secrets Manager secret associated with a pipeline source repository."
 	svcPortFlagDescription           = "Optional. The port on which your service listens."
+
+	storageFlagDescription        = "Name of the storage resource to create."
+	storageServiceFlagDescription = "Name of the service to associate with storage."
 )
 
+var storageTypeFlagDescription = fmt.Sprintf(`Type of storage to add. Must be one of:
```

Comment:
nit: There is a section at l. 70 for variable descriptions, if you wouldn't mind moving the var there.